### PR TITLE
fix some ui issues according to comments

### PR DESCRIPTION
**Pull request by sanjanadhikari1**
- Fixed the messed up padding in members & portfolio page
- change the text "start wallet node: waiting for pasteld to be started" to "waiting for the pastel Damon to start"
- change text "Airdrop Promocode" to "Pastel Promocode"and make it the first option
- https://blockchain-devel.slack.com/archives/C01RADG7L7L/p1630097134061200

### CODE REVIEW

File: src/features/onboarding/register/StepPayment.tsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ const StepPaymentMethod = (props: TStepPaymentMethodProps): JSX.Element => {
       method: PaymentMethods.PslAddress,
     },
     {
-      name: 'Airdrop “Promo Code”',
+      name: 'Pastel Promocode',
       method: PaymentMethods.AirdropPromoCode,
```
Comment (CrazyOptimist) [inline src/features/onboarding/register/StepPayment.tsx:35]: name is Pastel Promocode, but method is still AirdropPromoCode - seems to be a mistake.
@ngvtuan is working on Pastel promo code in parallel pull request, so I suggest to just undo this change

File: src/features/profile/components/MyProfileGeneral/NSFWControls.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ export default function NSFWControls(): JSX.Element {
   return (
     <div className='w-full mt-20 mb-50px'>
       <div className='flex pt-3'>
-        <div className='text-gray-71 mr-4'>Hide Not Safe For Work NFTs</div>
+        <div className='text-gray-71 mr-4'>Show advanced features</div>
```
Comment (CrazyOptimist) [inline src/features/profile/components/MyProfileGeneral/NSFWControls.tsx:24]: Text is changed, but implementation remains the same, I think it's not correct. Could you change implementation as well, so "Hide Not Safe For Work" will be toggled by clicking on "Show advanced features"?

**Review by CrazyOptimist — CHANGES_REQUESTED** [review]

File: src/features/profile/components/MyProfileGeneral/NSFWControls.tsx (hunk 2 above)
Comment (sanjanadhikari1) [reply src/features/profile/components/MyProfileGeneral/NSFWControls.tsx]: I have changed implementation as well, not only text.
Please check again.
! [image](https://user-images.githubusercontent.com/64049500/131484635-9b5d479b-ea8a-495a-b6c3-3d038a8212a6.png)

**Review by CrazyOptimist — APPROVED** [review]

**Review by ngvtuan — APPROVED** [review]

**Review by CrazyOptimist — APPROVED** [review]
